katello.te

```diff
@@ -55,6 +55,9 @@ allow tomcat_t katello_candlepin_port_t:tcp_socket name_connect;
 # Candlepin Event Listener connects to Artemis
 allow foreman_rails_t candlepin_activemq_port_t:tcp_socket name_connect;
 
+# Katello connection to qpid for katello agent
+allow foreman_rails_t amqp_port_t:tcp_socket name_connect;
```

Comment:
Does `amqp_port_t` need to be added to the require block?

Reply:
Yep, fixed

Reply:
You could have written `corenet_tcp_connect_amqp_port(foreman_rails_t)` but this will do.